src/main/java/com/practicesoftwaretesting/cart/controller/CartController.java

```diff
@@ -0,0 +1,31 @@
+package com.practicesoftwaretesting.cart.controller;
+
+import com.practicesoftwaretesting.common.BaseController;
+import com.practicesoftwaretesting.cart.model.AddCartItemRequest;
+import io.restassured.response.Response;
+
+public class CartController extends BaseController {
+
+    private static final String CARTS = "/carts/";
```

Comment:
i don't recommend to keep the path in constants, for me this approach has lack of readability. 
In case where you use "/carts/" inside the post or get method of the RestAssured client it does not look as a magic number, so I don't think you should move it to constants.
And you left "/carts" without moving to constants, so it looks a bit non-consistent.

Reply:
I have inlined constant value.